sdk/iot/hub/samples/src/generate_certificate.sh

```diff
@@ -2,11 +2,13 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # SPDX-License-Identifier: MIT
 
+set -o nounset # Exit if variable not set.
+set -o pipefail # Exit if pipe failed.
+
 openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
 openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"
 openssl x509 -noout -text -in device_ec_cert.pem
 
-
 rm device_cert_store.pem
```

Comment:
you could use rm -f device_cert_store.pem

Reply:
done